src/google-classroom-extractor/google_classroom_extractor/api/courses.py

```diff
@@ -0,0 +1,40 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+import logging
+from typing import List, Dict
+import pandas as pd
+from .api_caller import call_api
+
+
```

Comment:
feels so weird to me not to have a class here. I haven't yet gotten used to that possibility.

Reply:
It looks similar to javascript when you just export functions from a file

Reply:
yes, same as JS where a module provides organizational benefit and you don't need state or multiple instances